components/breadcrumb/index.jsx

```diff
@@ -20,20 +20,56 @@ import componentDoc from './docs.json';
 
 // ## Constants
 import { BREADCRUMB } from '../../utilities/constants';
+import Dropdown from './../menu-dropdown';
+
+const propTypes = {
+	/**
+	 * **Assistive text for accessibility.**
+	 * This object is merged with the default props object on every render.
+	 * * `label`: The assistive text for the breadcrumb trail.
+	 */
+	assistiveText: PropTypes.shape({
+		label: PropTypes.string,
+	}),
+	/**
+	 * A unique ID is needed in order to support keyboard navigation, ARIA support, and connect the dropdown to the triggering button.
+	 */
+	id: PropTypes.string,
+	/**
+	 * Overflow menu of the type [Dropdown](/components/menu-dropdowns)
+	 */
+	overflowDropdownMenu: PropTypes.node,
+	/**
+	 * An array of react elements presumably anchor elements.
```

Comment:
Maybe we could say: "An array of anchor elements that define the path to the current record."